ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/package.scala

```diff
@@ -42,4 +43,11 @@ package object kvutils {
 
   implicit val `Bytes Ordering`: Ordering[Bytes] = Ordering.by(_.asReadOnlyByteBuffer)
 
+  implicit object `DamlStateValue with Fingerprint has DamlStateValue`
+      extends HasDamlStateValue[(Option[DamlStateValue], Fingerprint)] {
```

Comment:
Would it make sense to add a type alias for this to improve readability?

Reply:
Gonna delete it shortly, so no thanks. :upside_down_face: